luigi/worker.py

```diff
@@ -447,17 +479,17 @@ def __init__(self, scheduler=None, worker_id=None, worker_processes=1, assistant
         if scheduler is None:
             scheduler = Scheduler()
 
+        self._config = worker(**kwargs)
+
+        assert self._config.wait_interval >= _WAIT_INTERVAL_EPS, "[worker] wait_interval must be positive"
+        assert self._config.wait_jitter >= 0.0, "[worker] wait_jitter must be equal or greater than zero"
+
```

Comment:
Why the need to move this code?

Reply:
(feel free to if you have a reason, I'm mostly curious)

Reply:
This was required as `_generate_worker_info()` needs to access `_config`.